glean-core/metrics.yaml

```diff
@@ -391,3 +391,17 @@ glean.upload:
     expires: never
     no_lint:
       - COMMON_PREFIX
+
+glean.database:
+  size:
+    type: memory_distribution
+    description: >
+      The size of the database file at startup.
+    memory_unit: byte
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1656589
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1656589#c7
+    notification_emails:
+      - glean-team@mozilla.com
+    expires: "2020-09-30"
```

Comment:
Should the time horizon be so short?  Maybe we do like 3 months to make sure it has plenty of lead time on the main Fenix channel?

Reply:
No, it's actually never expiring. Forgot to update the PR